src/service/repo/signedMessageRepo.ts

```diff
@@ -45,7 +45,7 @@ interface SignedMessageRepoTypes {
     attachments: any,
     replyMessageId?: string
   ) => Promise<any>;
-  getAllSignedChannels: (timeStamp: string) => Promise<ChannelData[]>;
+  getAllSignedChannels: (timeStamp?: string) => Promise<ChannelData[]>;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change made to the `getAllSignedChannels` function signature, making the `timeStamp` parameter optional, is a good one. This allows for more flexibility when calling the function, as it can now be called without a timestamp if necessary.
```diff
- getAllSignedChannels: (timeStamp: string) => Promise<ChannelData[]>;
+ getAllSignedChannels: (timeStamp?: string) => Promise<ChannelData[]>;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->